integration-test/src/test/java/org/apache/iotdb/db/it/aggregation/IoTDBAggregationByLevelIT.java

```diff
@@ -674,6 +675,25 @@ public void TestGroupByLevelWithoutAggregationFunc() {
     }
   }
 
+  @Test
+  public void groupByLevelWithSameColumn() throws SQLException {
+    String[] retArray = new String[] {"8", "8"};
+    try (Connection connection = EnvFactory.getEnv().getConnection();
+        Statement statement = connection.createStatement()) {
+
+      int cnt = 0;
+      try (ResultSet resultSet =
+          statement.executeQuery(
+              "select count(status),count(status) from root.*.* GROUP BY level=0")) {
+        while (resultSet.next()) {
+          String ans = resultSet.getString(count("root.*.*.status"));
```

Comment:
It seems that this case can't verify whether the output has 2 or 1 columns. You should use `resultSet.getMetadata()` and check
1. It contains 2 columns.
2. The 1st and 2nd column name are both `count(root.*.*.status)`.

Reply:
`String[] retArray = new String[] {"8", "8"};`

The results that already contain two columns should be changed here

Reply:
Thanks. Maybe the while loop can be removed, and add another assert `assertEquals(count("root.*.*.status"), metadata.getColumn(1)); `

Reply:
ok~~